cmd/podman/registry/config.go

```diff
@@ -68,22 +68,23 @@ func newPodmanConfig() {
 		}
 	}
 
-	// FIXME: for rootless, add flag to get the path to override configuration
-	cfg, err := config.NewConfig("")
+	cfg, err := config.ReadCustomConfig()
```

Comment:
This should not change.  This eliminates all of the defaults and reads only a custom config.  It should never be done for the global setting, only when you are customising the local.
This is why SELinux is breaking.